doc/lang.md

```diff
@@ -557,16 +539,15 @@ Quint is not Fashion Police.
 
 ### Imports
 
-As it may be tedious to access definitions via the dot-notation, Quint supports
-name imports:
+Similar to mainstream programming languages, Quint supports name imports:
 
-```scala
-module Imports {
-  module Math {
-    def sqr(x) = x * x
-    def pow(x, y) = x^y
-  }
+```bluespec
+module Math {
+  def sqr(x) = x * x
+  def pow(x, y) = x^y
+}
 
+module Imports {
   // import 'pow' from Math
   import Math.pow
 
```

Comment:
Bit inconsistent that this uses `.` instead of `::` to qualify the name, no?

Reply:
I meant: why `Math.pow` instead of `Mat::pow`?

Reply:
This is actually a very good point. @bugarela wdyt? I am actually not even sure about why we would need `::` anymore. It could be properly resolved at the name resolution phase, as in many programming languages. I would assume that `import Math::pow` is a bit unconventional.

Reply:
I also find this inconsistency jarring (and noted it somewhere). It also shows up in the unqualified import syntax: ‘import A.*`.

I don’t think `import Math::pow` unconventional? I know C++ doesn’t support importing single functions, but Rust, which does, uses `::` consistently as the module accessor: https://doc.rust-lang.org/reference/items/use-declarations.html

And this follows all other languages I know of that use the same `.` syntax for qualified reference and for specific import (e.g., Scala, Python, Ocaml, Haskell, JavaScript).

Reply:
Thanks @shonfeder! I keep forgetting about rust. Since there is a precendent in the form of rust, we could of migrate all of our import syntax to `::`. It should be much easier to implement than disambiguation of `.` with a name resolver.